src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  margin: 0;
+  
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
```

Comment:
Read more about normalize.css and reset.css files

https://www.geeksforgeeks.org/difference-between-resetting-and-normalizing-in-css/

https://stackoverflow.com/questions/6887336/what-is-the-difference-between-normalize-css-and-reset-css